database-main-reader/src/main/java/com/dealflowbus/databasemainreader/models/Lead.java

```diff
@@ -33,216 +32,229 @@
+    @JoinColumn(name = "lead_id2")
+    private List<DBFile> files;
 
+    public Lead() {
+
+    }
+
+    public Lead(String projectName, String projectOwner,
+            String field, String email, String extraAddress) {
+
+        this.projectName = projectName;
+        this.projectOwner = projectOwner;
+        this.inProgress = false;
+        this.rejected = false;
+        this.inPortfolio = false;
+        this.detail = new Detail();
+        this.field = field;
+        this.email = email;
+        this.extraAddress = extraAddress;
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public String getProjectName() {
+        return projectName;
+    }
+
+    public void setProjectName(String projectName) {
+        this.projectName = projectName;
+    }
+
+    public String getProjectOwner() {
+        return projectOwner;
+    }
+
+    public void setProjectOwner(String projectOwner) {
+        this.projectOwner = projectOwner;
+    }
+
+    public boolean isInProgress() {
+        return inProgress;
+    }
+
+    public void setInProgress(boolean inProgress) {
+        this.inProgress = inProgress;
+    }
+
+    public boolean isRejected() {
+        return rejected;
+    }
+
+    public void setRejected(boolean rejected) {
+        this.rejected = rejected;
+    }
+
+    public Detail getDetail() {
+        return detail;
+    }
+
+    public void setDetail(Detail detail) {
+        this.detail = detail;
+    }
+
+    public String getField() {
+        return field;
+    }
+
+    public void setField(String field) {
+        this.field = field;
+    }
+
+    public LocalDate getDateArrival() {
+        return dateArrival;
+    }
+
+    public void setDateArrival(LocalDate dateArrival) {
+        this.dateArrival = dateArrival;
+    }
+
+    public LocalDate getLastTouched() {
+        return lastTouched;
+    }
+
+    public void setLastTouched(LocalDate lastTouched) {
+        this.lastTouched = lastTouched;
+    }
+
+
+    public List<Note> getNotes() {
+        return notes;
+    }
+
+    public void setNotes(List<Note> notes) {
+        this.notes = notes;
+    }
+
+    public void addNote(Note note) {
+        if (notes == null) {
+            notes = new ArrayList<Note>();
+        }
+        notes.add(note);
+    }
+
+    public boolean isInPortfolio() {
+        return inPortfolio;
+    }
+
+    public void setInPortfolio(boolean inPortfolio) {
+        this.inPortfolio = inPortfolio;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getExtraAddress() {
+        return extraAddress;
+    }
+
+    public void setExtraAddress(String extraAddress) {
+        this.extraAddress = extraAddress;
+    }
+
+    public void addFile(DBFile file) {
+        if (files == null) {
+            files = new ArrayList<DBFile>();
```

Comment:
to constructor please.

Reply:
To the default empty constructor or to the constructor with parameters?

Reply:
done! :)